command/ibft/ibft_init.go

```diff
@@ -56,6 +56,19 @@ func (p *IbftInit) GetBaseCommand() string {
 	return "ibft init"
 }
 
+// generateAlreadyInitializedError generates an output for when the IBFT directory
+// has already been initialized in the past
+func generateAlreadyInitializedError(directory string) string {
+	output := "\n[IBFT INIT ERROR]\n"
+	output += fmt.Sprintf("Directory %s has previously initialized IBFT data\n", directory)
+	return output
+}
+
+var (
+	CONSENSUS_DIR = "consensus"
+	LIBP2P_DIR    = "libp2p"
+)
```

Comment:
Just small suggestions
(1) In Golang, name of constant variable is usually named by CamelCase style. (If you don't want to export constant, then can use lowerCamelCase)
https://golang.org/doc/effective_go#constants
https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const

(2) `"consensus"` and `"libp2p"` are literals, so you can use `const` instead of `var`.

Reply:
You're right, completely forgot about this 👍 

Fixed 👍 

f684a99